src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,111 @@
+@use '../utils/variables' as *;
+@use '../utils/mixins' as *;
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding: 16px;
+
+  font-family: Roboto, sans-serif;
+
+  @include transition;
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+
+    margin-top: $margin-top;
+  }
+
+  &__item-model {
+    font-weight: $medium-font-weight;
```

Comment:
The `$medium-font-weight` variable is used here. Ensure that it is defined in the `variables` file to avoid any compilation errors.